libyul/backends/evm/EVMCodeTransform.cpp

```diff
@@ -728,7 +723,8 @@ size_t CodeTransform::variableHeightDiff(Scope::Variable const& _var, YulString
 			to_string(heightDiff - limit) +
 			" slot(s) too deep inside the stack."
 		);
-		BOOST_THROW_EXCEPTION(m_stackErrors.back());
+		m_assembly.markAsInvalid();
+		return 0;
```

Comment:
Maybe at least return 1 so that it will work for `dup`?

Reply:
Actually there's no real need - all call sites handle 0 gracefully... but maybe still better to change it.
On the other hand there's e.g. this:
```
		if (size_t heightDiff = variableHeightDiff(_var, _variableName.name, true))
			m_assembly.appendInstruction(evmasm::swapInstruction(heightDiff - 1));
```
So returning ``1`` would actually cause problems...
I'll change it to return ``_forSwap ? 2 : 1``, that should work...